metaci/conftest.py

```diff
@@ -1,51 +1,107 @@
 import factory
 import factory.fuzzy
 
+import numbers
+import random
+
 from metaci.plan.models import Plan, PlanRepository
 from metaci.testresults.models import TestResult, TestMethod, TestClass
-from metaci.build.models import BuildFlow, Build
+from metaci.build.models import BuildFlow, Build, BUILD_STATUSES, BUILD_FLOW_STATUSES
 from metaci.repository.models import Branch, Repository
 
 from metaci.users.models import User
 
+BUILD_STATUS_NAMES = (name for (name, label) in BUILD_STATUSES)
+BUILD_FLOW_STATUS_NAMES = (name for (name, label) in BUILD_FLOW_STATUSES)
+
+
+def do_logs():
+    import logging
+
+    logger = logging.getLogger("factory")
+    logger.addHandler(logging.StreamHandler())
+    logger.setLevel(logging.DEBUG)
+
+
+def fake_name(prefix=None):
+    return factory.LazyAttribute(
+        lambda a: (getattr(a, "_name_prefix", None) or prefix or "")
+        + factory.Faker("word").generate({})
+    )
+
+
+class RelatedFactoryList(factory.RelatedFactory):
+    """Calls a factory 'size' times once the object has been generated.
+        Simplified from
+        https://github.com/FactoryBoy/factory_boy/blob/2481d411cde311bb7edd51b6dff3b345c2c14bc6/factory/declarations.py#L675
+        While awaiting release
```

Comment:
?

Reply:
Sorry to be unclear. I'll fix that comment too. The method is described in the docs for FactoryBoy but when I tried to use it, it wasn't there. It turns out that FactoryBoy has been awaiting release for about 10 months and the documentation is more up-to-date than the releases.

But that said: I ended up solving my problem another way, so I'll delete this class.

Reply:
https://github.com/SFDO-Tooling/MetaCI/pull/408/commits/cb9ac02e086d411c208d84d35e2119e5e8efb7bd